zuul.d/jobs.yaml

```diff
@@ -158,24 +158,32 @@
       ansible_collections_repo: "{{ zuul.project.canonical_name }}"
 
 - job:
-    name: ansible-test-units-netcommon-python27
-    parent: ansible-test-units-base-python27
+    name: ansible-test-units-netcommon-python36
+    parent: ansible-test-units-base-python36
     required-projects:
       - name: github.com/ansible-collections/ansible.netcommon
```

Comment:
`  - name: github.com/ansible/ansible
        override-checkout: stable-2.9
`

needs to be added